associations/views.py

```
@@ -102,103 +101,161 @@ def confirm(request, con_num):
 
 def home(request):
     user = request.user
-    gitTics = ''
-    zenTics = ''
+    git_tics = ''
+    zen_tics = ''
 
     try:
-        github = Github(username=user.git_name, 
-                    api_token=user.git_key)
-        repo = user.git_repo
+        r_op = requests.get('https://api.github.com/users/%s/%s/issues' %
+                            (user.git_org, user.git_repo),
+                            params={'state': 'open'},
+                            auth=(user.git_name, user.git_pass))
+        r_cl = requests.get('https://api.github.com/users/%s/%s/issues' %
+                            (user.git_org, user.git_repo),
+                            params={'state': 'closed'},
+                            auth=(user.git_name, user.git_pass))
+        
+        gopen_list = r_op.json
+        gclosed_list = r_cl.json
+
+        if type(r_op.text) is type({}) and "message" in r_op.text:
+            git_tics = 'broken'
     except:
```

What exception are we trying to catch here? If possible, we should use specific exceptions so that unexpected things aren't accidentally silently swallowed.